### PR TITLE
Proofreading

### PR DESCRIPTION
**Pull request by TonyWhitley**

### CODE REVIEW

File: Source/rFactor2SharedMemoryMap.cpp [hunk 1]
```diff
@@ -39,12 +39,12 @@ Shared resources:
...
     * PluginControl - mapped view of rF2PluginControl structure
 
-  Aside from Extended (see below), output buffers are (with few exceptions) exact mirror of ISI structures, plugin constantly memcpy'es them
+  Aside from Extended (see below), output buffers are (with few exceptions) exact mirror of ISI structures, plugin constantly memcpy's them
   from game to memory mapped files.
 
   Plugin offers optional weak synchronization by using version variables on each of the output buffers.
 
-  Input buffers are meant to be filled out by the clients.  To avoid complex locking input buffers use version variables as well, and were
+  Input buffers are meant to be filled out by the clients.  To avoid complex locking input buffers use they version variables as well, and were
```
Comment (TheIronWolfModding) [inline Source/rFactor2SharedMemoryMap.cpp:47]: their?

Comment (TonyWhitley) [reply Source/rFactor2SharedMemoryMap.cpp:47]: Perhaps, probably, maybe.  I could offer arguments both ways, go with their.

Comment (TheIronWolfModding) [reply Source/rFactor2SharedMemoryMap.cpp:47]: Just triple checking - is "they" in the right place in a sentence?

Comment (TonyWhitley) [reply Source/rFactor2SharedMemoryMap.cpp:47]: Oops.  ..._they use version_...

**Pull request merged**